### ISSUE TITLE
Ombi with Oauth enabled and App Access

### ISSUE THREAD

**Issue by IrishTR** [htpcBeginner/docker-traefik#221]
Hi, first off your work has enabled me to accomplish a great many things with my setup, thank you very much.

On to my question I saw previous issue where folks needed to bypass Authelia/Google OAuth for API access for things like Tautulli, Radarr, Sonarr, etc.  I've got all that working great.  My last hold out is the Ombi piece.  Not sure what needs to happen to have OAuth enable and allow API and or the Mobile App to still function (similar to Tautulli).  I tried the API bypass options used for those other apps listed but don't seem to be making headway and not sure where to begin tracking down if its something I need to tweak or just not feasible.

Thank you.

**Comment by IrishTR** [comment]
`  ombi:
    image: linuxserver/ombi:latest
    container_name: ombi
    restart: unless-stopped
    networks:
      - t2_proxy
#    ports:
#      - "$OMBI_PORT:3579"
    security_opt:
      - no-new-privileges:true
    volumes:
      - $DOCKERDIR/ombi:/config
    environment:
      PUID: $PUID
      PGID: $PGID
      TZ: $TZ
      BASE_URL: /ombi #optional
    labels:
      - "traefik.enable=true"
      - "traefik.http.routers.ombi-rtr.entrypoints=https"
      - "traefik.http.routers.ombi-rtr.priority=99"
      - "traefik.http.routers.ombi-rtr.rule=Host(`ombi.$DOMAINNAME`)"
      - "traefik.http.routers.ombi-rtr.middlewares=chain-oauth@file"
      - "traefik.http.routers.ombi-rtr.service=ombi-svc"
      - "traefik.http.services.ombi-svc.loadbalancer.server.port=3579"
      ## API Oauth Bypass
      - "traefik.http.routers.ombi-rtr-bypass.entrypoints=https"
      - "traefik.http.routers.ombi-rtr-bypass.rule=Headers(`X-Api-Key`, `$OMBI_API`) || Query(`apikey`, `$OMBI_API`)"
      - "traefik.http.routers.ombi-rtr-bypass.priority=100"
      - "traefik.http.routers.ombi-rtr-bypass.middlewares=chain-no-auth@file"
      - "traefik.http.routers.ombi-rtr-bypass.service=ombi-svc"`

**Comment by IrishTR** [comment]
ombi:
    image: linuxserver/ombi:latest
    container_name: ombi
    restart: unless-stopped
    networks:
      - t2_proxy
    security_opt:
      - no-new-privileges:true
    volumes:
      - $DOCKERDIR/ombi:/config
    environment:
      PUID: $PUID
      PGID: $PGID
      TZ: $TZ
      BASE_URL: /ombi #optional
    labels:
      - "traefik.enable=true"
      - "traefik.http.routers.ombi-rtr.entrypoints=https"
      - "traefik.http.routers.ombi-rtr.priority=99"
      - "traefik.http.routers.ombi-rtr.rule=Host(`ombi.$DOMAINNAME`)"
      - "traefik.http.routers.ombi-rtr.middlewares=chain-no-auth@file"
      - "traefik.http.routers.ombi-rtr.service=ombi-svc"
      - "traefik.http.services.ombi-svc.loadbalancer.server.port=3579"
      ## API Oauth Bypass
       - "traefik.http.routers.ombi-rtr-bypass.entrypoints=https"
       - "traefik.http.routers.ombi-rtr-bypass.rule=Headers(`X-Api-Key`, `$OMBI_API`) || Query(`apikey`, `$OMBI_API`)"
       - "traefik.http.routers.ombi-rtr-bypass.priority=100"
       - "traefik.http.routers.ombi-rtr-bypass.middlewares=chain-no-auth@file"
       - "traefik.http.routers.ombi-rtr-bypass.service=ombi-svc"

**Comment by robflate** [comment]
Haven't checked on this in a while but Traefik used to throw errors and quietly fail when adding those headers to Ombi.

Not as secure but you could bypass;

```
- "traefik.http.routers.ombi-rtr-bypass.rule=Host(`ombi.$DOMAINNAME`) && PathPrefix(`/api/`)"
```

**Comment by uasdj25** [comment]
Yep, I was told on Ombi discord a while back that it wasn't feasible with Google Oauth.  There is, however, the ability to use Plex Oauth that is embedded within the app.  I haven't gone back, so this might not be the case anymore (the inability to leverage oauth from Google through Traefik), but to my knowledge not doable.  I have it just using oauth from Plex.
